Packs/Orca/Integrations/Orca/Orca.py

```diff
@@ -237,6 +315,108 @@ def fetch_incidents(
     return next_run, incidents
 
 
+def set_alert_severity(orca_client: OrcaClient, args: Dict[str, Any]) -> CommandResults:
+    alert_id = args.get("alert_id")
+    score = args.get("score")
+
+    assert alert_id and score
+    demisto.debug(f"Set alert severity {alert_id=} {score=}")
+
+    response = orca_client.set_alert_score(alert_id=alert_id, orca_score=score)
+
+    context = {}
+    if "alert_id" in response:
+        context = {
+            "id": response.get("alert_id"),
+            "details": response.get("details", {}).get("description"),
+            "severity": response.get("details", {}).get("severity")
+        }
+    if "error" in response:
+        raise DemistoException(response.get("error"))
+
+    return CommandResults(
+        readable_output=f"Alert severity changed to {score}",
+        outputs_prefix="Orca.Alert",
+        outputs=context,
+        raw_response=response
+    )
+
+
+def get_alert_event_log(orca_client: OrcaClient, args: Dict[str, Any]) -> CommandResults:
+    alert_id = args.get("alert_id")
+    limit = cast(int, args.get("limit", 20))
+    start_at_index = cast(int, args.get("start_at_index", 0))
+    event_log_type = args.get("type")
+
+    demisto.debug(f"Get alert event log {alert_id=} {limit=} {start_at_index=} {event_log_type=}")
+
+    assert alert_id
+
+    response = orca_client.get_alert_event_log(
+        alert_id=alert_id,
+        limit=limit,
+        start_at_index=start_at_index,
+        event_log_type=event_log_type
+    )
+    context = response.get("event_log", [])
+
+    alert_logs = [{
+        "id": item.get("id"),
+        "alert_id": item.get("alert_id"),
+        "type": item.get("type"),
+        "description": item.get("details", {}).get("description")
+    } for item in context]
+
+    return CommandResults(
+        readable_output=tableToMarkdown(f"Alert event log ({alert_id})", alert_logs, removeNull=True),
```

Comment:
did you check if we're getting results when running the command?

Reply:
Yes, sure
<img width="1537" alt="image" src="https://user-images.githubusercontent.com/90267674/236782404-b5c11ad6-a256-41ba-82f5-d272d5f2652e.png">